pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1340,77 +1339,130 @@ func (n *NetworkPolicyController) getClusterGroupWorkloads(group *antreatypes.Gr
 	return pods, ees
 }
 
+func getInternalNetworkPolicyName(ref controlplane.NetworkPolicyReference) string {
+	return string(ref.UID)
+}
+
 // syncInternalNetworkPolicy retrieves all the AppliedToGroups associated with
 // itself in order to calculate the Node span for this policy.
-func (n *NetworkPolicyController) syncInternalNetworkPolicy(key string) error {
+func (n *NetworkPolicyController) syncInternalNetworkPolicy(key controlplane.NetworkPolicyReference) error {
+	internalNetworkPolicyName := getInternalNetworkPolicyName(key)
+	klog.V(2).Infof("Syncing internal NetworkPolicy %s", key)
```

Comment:
ditto and other places.

Reply:
actually these are existing logs, they appear as diff somehow because of move of some lines.
I would try to change them to structured logging since they are pointed out, but I may not do further log cleanup as it's not the purpose of the PR and would increase the burdon of review given it already adds 1000+ lines and removes 2000+ lines.

Reply:
Make sense.